### PR TITLE
resource_sql keyword filter fix and iopro sql->ndarray

### PR DESCRIPTION
**Pull request by nevermindewe**

### CODE REVIEW

File: into/backends/sql.py [hunk 1]
```diff
@@ -60,6 +60,15 @@
                  sa.types.Float: 'float64'})
 
 
+sa_create_engine_kwargs=['case_sensitive', 'connect_args', 'convert_unicode',
```
Comment (cpcloud) [inline into/backends/sql.py]: small style nit: spaces around `=`

**Comment by nevermindewe** [comment]
I completely forgot about my earlier PR, so I didn't make a new branch when doing the iopro stuff.

**Comment by cpcloud** [comment]
@nevermindewe is this superceded by the other iopro PR #148

**Comment by cpcloud** [comment]
if so, please close

**Comment by nevermindewe** [comment]
superceded by PR #148

**Pull request closed without merging**